app/models/organization_wizzard.rb

```diff
@@ -0,0 +1,31 @@
+class OrganizationWizzard
+  include Rails.application.routes.url_helpers
+
+  def initialize(organization)
+    @organization = organization
+  end
+
+  def ready?
+    have_account? && have_categories?
+  end
+
+  def not_ready?
```

Comment:
@shir the name of the method is not so important, it may have name `continues?`. But the idea of this method is very important. Example from rails is `model.persisted?` and `model.new_object?`.